internal/mock/sqladmin.go

```diff
@@ -118,7 +118,7 @@ func InstanceGetSuccess(i FakeCSQLInstance, ct int) *Request {
 	db := &sqladmin.DatabaseInstance{
 		BackendType:     "SECOND_GEN",
 		ConnectionName:  fmt.Sprintf("%s:%s:%s", i.project, i.region, i.name),
-		DatabaseVersion: i.dbVersion,
+		DatabaseVersion: i.version,
```

Comment:
`db` is a useful qualifier on version for me

Reply:
:+1: